### PR TITLE
Modifications in data storage 

### PR DESCRIPTION
**Pull request by dalabayev**
-Added needed data to save
-Modified json file structure
-Modified logic to save the coordinates

### CODE REVIEW

File: gazeplay-commons/lombok.config [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-commons/lombok.config]: Can you revert the changes on this file? Nothing appears to have changed.

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java [hunk 2]
```diff
@@ -231,19 +241,22 @@ public void start() {
                 }
                 if (config.getAreaOfInterestDisabledProperty().getValue()) {
                     if (getX != previousX || getY != previousY) {
-                        final long timeToFixation = System.currentTimeMillis() - startTime;
+                        //final long timeToFixation = System.currentTimeMillis() - startTime;
                         previousX = getX;
                         previousY = getY;
-                        final long timeInterval = (timeToFixation - previousTime);
+                        //final long timeInterval = (timeToFixation - previousTime);
```
Comment (Tohaker) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java]: Instead of commenting code, we'd prefer to delete it. If there are reasons to only temporarily comment this code, please add an explanation comment.

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java [hunk 3]
```diff
@@ -252,10 +265,10 @@ public void start() {
                 }
                 if (config.getAreaOfInterestDisabledProperty().getValue()) {
                     if (getX != previousX || getY != previousY && counter == 2) {
-                        final long timeElapsedMillis = System.currentTimeMillis() - startTime;
+                        //final long timeElapsedMillis = System.currentTimeMillis() - startTime;
                         previousX = getX;
                         previousY = getY;
-                        final long timeInterval = (timeElapsedMillis - previousTime);
+                        //final long timeInterval = (timeElapsedMillis - previousTime);
```
Comment (Tohaker) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java]: Same as above.

File: gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java [hunk 4]
```diff
@@ -343,9 +358,12 @@ public SavedStatsInfo saveStats() throws IOException {
         g.setColor(Color.BLACK);
         g.fillRect(0, 0, bImage.getWidth(), bImage.getHeight());
         g.drawImage(screenshotImage, 0, 0, null);
+        BufferedWriter bf = new BufferedWriter(new FileWriter(replayDataFile));
```
Comment (Tohaker) [inline gazeplay-commons/src/main/java/net/gazeplay/commons/utils/stats/Stats.java]: Please put this into a try-with-resource block, so the BufferedWriter is closed automatically.

File: gazeplay-dist/lombok.config [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-dist/lombok.config]: Please revert this file

File: gazeplay-games-commons/lombok.config [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-games-commons/lombok.config]: Please revert this file

File: gazeplay-games/lombok.config [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-games/lombok.config]: Please revert this file

File: gazeplay-melordi/lombok.config [hunk 8]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-melordi/lombok.config]: Please revert this file

File: gazeplay-picto-pick/lombok.config [hunk 9]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-picto-pick/lombok.config]: Please revert this file

File: gazeplay-side-projects/lombok.config [hunk 10]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay-side-projects/lombok.config]: Please revert this file

File: gazeplay/lombok.config [hunk 11]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline gazeplay/lombok.config]: Please revert this file

File: lombok.config [hunk 12]
```diff
@@ -1,4 +1,4 @@
-# This file is generated by the 'io.freefair.lombok' Gradle plugin
-config.stopBubbling = true
-lombok.addLombokGeneratedAnnotation = true
-lombok.extern.findbugs.addSuppressFBWarnings = true
+# This file is generated by the 'io.freefair.lombok' Gradle plugin
+config.stopBubbling = true
+lombok.addLombokGeneratedAnnotation = true
+lombok.extern.findbugs.addSuppressFBWarnings = true
```
Comment (Tohaker) [inline lombok.config]: Please revert this file

File: gazeplay-commons/lombok.config (hunk 1 above)
Comment (dalabayev) [reply gazeplay-commons/lombok.config:4]: Could you pleae help me? I don't know how to revert this file. I've tried to do it but since nothing have changed I couldn't revert it
